### PR TITLE
Add entity models

### PR DESCRIPTION
**Pull request by jackieHubc**
I make new entity models for Receiver, Sender, Restaurant, Food, Deliver, Rating.
Some changes may be needed later.

### CODE REVIEW

File: src/main/java/com/feedr/dao/OrderDAO.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+import com.feedr.util.DatabaseConnector;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import java.sql.SQLException;
+import java.sql.Timestamp;
+import java.sql.Date;
+
+
+public class OrderDAO {
```
Comment (omzmarlon) [inline src/main/java/com/feedr/dao/OrderDAO.java]: Next time, please put OrderDAO into a separate PR. This PR is for entity models.
Catherine still have new changes for order model, so this class will have to change

File: src/main/java/com/feedr/models/DeliverModel.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
+package com.feedr.models;
+import java.sql.Timestamp;
+
+public class DeliverModel {
```
Comment (omzmarlon) [inline src/main/java/com/feedr/models/DeliverModel.java]: Please follow java method/field naming convention. 
They are camel case not connected by underscore.
Update other classes method/field naming too

Comment (jackieHubc) [reply src/main/java/com/feedr/models/DeliverModel.java]: Fixed! Update soon

**Comment by omzmarlon** [comment]
Good job!
